src/com/facebook/buck/cli/Main.java

```diff
@@ -1786,18 +1786,24 @@ private static void daemonizeIfPossible() {
     IntByReference master = new IntByReference();
     IntByReference slave = new IntByReference();
 
-    openPtyLibrary.openpty(master, slave, Pointer.NULL, Pointer.NULL, Pointer.NULL);
+    int success = openPtyLibrary.openpty(master, slave, Pointer.NULL, Pointer.NULL, Pointer.NULL);
+    if (success != 0) {
```

Comment:
Seems good, but you don't need to create a temporary variable for the return value, can inline it?

Like

```
if (openPtyLibrary.openpty(master, slave, Pointer.NULL, Pointer.NULL, Pointer.NULL) != 0) {
  throw new RuntimeException("Failed to open pty");
}
```